tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -39,8 +39,19 @@ if (NOT WIN32)
   # Threads: defines CMAKE_THREAD_LIBS_INIT and adds -pthread compile option
   # for targets that link ${CMAKE_THREAD_LIBS_INIT}.
   find_package (Threads)
+
+  option(tensorflow_PATH_STATIC_LIB "Additional library search path for libcudnn_static.a, libnccl_static.a, libculibos.a" /usr/local/cuda/lib64/)
+  option(tensorflow_CUDNN_INCLUDE "cudnn.h header install path" /usr/include/)
 endif()
 
+if (WIN32)
+  set(BOOL_WIN32 ON)
```

Comment:
Do we need a separate `BOOL_WIN32` symbol when we already have `WIN32`?

Reply:
Is is because WIN32 cannot be used in '''<BOOL:${BOOL_WIN32}>:...'''